README.md

```diff
@@ -6,22 +6,19 @@ Rust userland library for Tock (WIP)
 
 This project is nascent and still under heavy development, but first steps:
 
-1. Ensure you have a working toolchain available:
+```bash
+./build_examples.sh
+```
 
-    `rustup install nightly-2017-12-16`
+Then load `blink` on a board:
 
-2. Get a copy of this toolchain, in this repo's root:
+```bash
+./run_example.sh blink
+```
 
-    `rustup override set nightly-2017-12-16`
-
-3. Need to grab a copy of the rust sources:
-
-    `rustup component add rust-src`
-
-4. Now you should be able to build with:
-
-    `xargo build --target thumbv7em-tock-eabi`
+### Note! Do not use rustc-1.26!
```

Comment:
I think you should not fix the version of the compiler at all. This should be managed by the toolchain file.